xgo-check.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Why it names `xgo-check.sh`, what does it check?

Reply:
It checks if package being tested actually can be build